packages/terra-clinical-result/src/common/other/_EnteredInError.jsx

```diff
@@ -36,9 +37,10 @@ const EnteredInError = (props) => {
       {...customProps}
       className={templateClassnames}
     >
-      <span aria-label={intl.formatMessage({ id: 'Terra.clinicalResult.statusInErrorAria' })}>
+      <span aria-hidden="true">
```

Comment:
Do we want to hide this? The implementation before implies that they wanted it to read the Terra.clinicalResult.statusInError message but also then the Terra.clinicalResult.statusInErrorAria message. 
It would read as "In error. Value uncharted as In Error." which I think makes sense - I can see the argument that maybe it's repetitive but I'm not sure if that's intentional or if we can skip it. Definitely need to double check this with Scott. 

The reason why I'm using aria-hidden for my jira is because '--' is never read out by the screen reader so it makes sense to hide that from the voice over.

Reply:
I checked with Scott this morning that only "Value uncharted as In Error" needs to be read out by the screen reader.
cc @scottwilmarth